src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -223,8 +229,12 @@
 
       <_WasmRuntimePackSrcFile ObjectFile="$(_WasmIntermediateOutputPath)%(FileName).o" />
 
-      <_DotnetJSSrcFile Include="$(_WasmRuntimePackSrcDir)\*.js" Exclude="$(_WasmRuntimePackSrcDir)\*.iffe.js"/>
+      <_WasmExtraJSFile Include="$(_WasmRuntimePackSrcDir)\*.extpre.js" Kind="extern-pre-js"/>
```

Comment:
I'm not sure if it belongs here, but a comment somewhere describing what each of these files (extpre.js, iffe.js, etc) actually does/is used for would be really valuable. I have no idea

Reply:
Added `src\mono\wasm\runtime\modularize-dotnet.md`, more feedback welcome!